member-service/src/main/java/com/hedvig/memberservice/aggregates/MemberAggregate.kt

```diff
@@ -0,0 +1,588 @@
+        apply(TrackingIdCreatedEvent(id, uuidGenerator.generateRandom()))
+    }
+
+    private fun formatName(name: String): String {
+        val lowercase = name.toLowerCase()
+        return Character.toUpperCase(lowercase[0]).toString() + lowercase.substring(1)
+    }
+
+    @Throws(RuntimeException::class)
+    private fun getPersonInformationFromBisnode(applyChain: ApplyMore?, ssn: String): ApplyMore? {
+        var chain = applyChain
+        logger.info("Calling bisnode for person information for {}", ssn)
+        val personList = bisnodeClient.match(ssn).persons
+        val person = personList[0].person
+        if (personList.size != 1) {
+            throw RuntimeException("Could not find person at bisnode.")
+        }
+        chain = chain!!.andThenApply { NameUpdatedEvent(id, person.preferredOrFirstName, person.familyName) }
+        val pi = BisnodeInformation(ssn, person)
+        if (pi.address.isPresent) {
+            chain = chain.andThenApply(
+                Supplier<Any> { LivingAddressUpdatedEvent(id, pi.address.get()) })
+        }
+        chain.andThenApply(Supplier<Any> { PersonInformationFromBisnodeEvent(id, pi) })
+        return chain
+    }
+
+    @CommandHandler
+    fun inactivateMember(command: InactivateMemberCommand?) {
+        if (status == MemberStatus.INITIATED || status == MemberStatus.ONBOARDING) {
+            apply(MemberInactivatedEvent(id))
+        } else {
+            val str = String.format("Cannot INACTIAVTE member %s in status: %s", id, status.name)
+            throw RuntimeException(str)
+        }
+    }
+
+    @CommandHandler
+    fun startOnboardingWithSSNCommand(command: StartOnboardingWithSSNCommand) {
+        if (status == MemberStatus.INITIATED || status == MemberStatus.ONBOARDING) {
+            apply(OnboardingStartedWithSSNEvent(id, command.ssn, SSNUpdatedEvent.Nationality.SWEDEN))
+            getBirthdateFromSwedishSsnAndApplyEvent(command.ssn, null)
+            apply(MemberStartedOnBoardingEvent(id, MemberStatus.ONBOARDING))
+        } else {
+            throw RuntimeException(String.format("Cannot start onboarding in state: %s", status))
+        }
+    }
+
+    private fun maybeApplyNameUpdatedEvent(firstName: String?, lastName: String?) {
+        val shouldUpdateFirstName = firstName != null && member.firstName != firstName
+        val shouldUpdateLastName = lastName != null && member.lastName != lastName
+
+        if (shouldUpdateFirstName || shouldUpdateLastName) {
+            apply(NameUpdatedEvent(
+                id,
+                if (shouldUpdateFirstName) firstName else member.firstName,
+                if (shouldUpdateLastName) lastName else member.lastName
+            ))
+        }
+    }
+
+    @CommandHandler
+    fun memberUpdateContactInformation(cmd: MemberUpdateContactInformationCommand) {
+        if (status == MemberStatus.SIGNED) {
+            logger.error("Will not update member info since member is SIGNED (memberId={})", id)
```

Comment:
If we want kotlin formatted strings instead
```suggestion
            logger.error("Will not update member info since member is SIGNED (memberId=$id)")
```